### ISSUE TITLE
PHP Fatal error:  Uncaught Error: Class 'PHP_CodeCoverage' not found in /tmp/coverage.humbug..

### ISSUE THREAD

**Issue by stefandoorn** [padraic/humbug#182]
Full error:

````
PHP Fatal error:  Uncaught Error: Class 'PHP_CodeCoverage' not found in /tmp/coverage.humbug.33d04dead2b20cc587e3232d5779daa6.cache:2
Stack trace:
#0 /home/vagrant/.composer/vendor/humbug/humbug/src/Utility/CoverageData.php(59): include()
#1 /home/vagrant/.composer/vendor/humbug/humbug/src/TestSuite/Mutant/Runner.php(140): Humbug\Utility\CoverageData->loadCoverageFor('/var/www/src/Ap...')
#2 /home/vagrant/.composer/vendor/humbug/humbug/src/TestSuite/Mutant/Runner.php(113): Humbug\TestSuite\Mutant\Runner->runBatch(Object(Humbug\TestSuite\Mutant\Collector), Object(Humbug\Utility\CoverageData), Array, NULL)
#3 /home/vagrant/.composer/vendor/humbug/humbug/src/Command/Humbug.php(135): Humbug\TestSuite\Mutant\Runner->run(Object(Humbug\Utility\CoverageData), Object(Humbug\MutableIterator), NULL)
#4 /home/vagrant/.composer/vendor/symfony/console/Command/Command.php(259): Humbug\Command\Humbug->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 /home/vagrant/.composer/v in /tmp/coverage.humbug.33d04dead2b20cc587e3232d5779daa6.cache on line 2
````

I've installed Humbug using the `composer global require` command and added it to PATH. Unit tests itself run all green.

Humbug config:

````
{
    "source": {
        "directories": [
            "src"
        ]
    },
    "chdir": "app",
    "timeout": 10,
    "logs": {
        "text": "humbuglog.txt"
    }
}
````

Project info: Symfony 2.8.7 with mainly CLI commands. Tests are mainly focussed on some stand alone classes.

**Comment by stefandoorn** [comment]
Output from before until the error:

````
$ humbug 

 _  _            _
| || |_  _ _ __ | |__ _  _ __ _
| __ | || | '  \| '_ \ || / _` |
|_||_|\_,_|_|_|_|_.__/\_,_\__, |
                          |___/ 
Humbug version 1.0-dev

Humbug running test suite to generate logs and code coverage data...

   60 [==========================================================] 4 secs

Humbug has completed the initial test run successfully.
Tests: 60 Line Coverage: 12.17%

Humbug is analysing source files...

Mutation Testing is commencing on 82 files...
(.: killed, M: escaped, S: uncovered, E: fatal error, T: timed out)

PHP Fatal error:  Uncaught Error: Class 'PHP_CodeCoverage' not found in /tmp/coverage.humbug.33d04dead2b20cc587e3232d5779daa6.cache:2
````

**Comment by stefandoorn** [comment]
Afterwards I installed it downloading the humbug.phar file using the wget method, then I don't get the errors above and all run fine.

**Comment by GrahamCampbell** [comment]
Looks like you;'re using a too new version of phpunit.

**Comment by stefandoorn** [comment]
Ah:

````
$ bin/phpunit --version
PHPUnit 5.4.6 by Sebastian Bergmann and contributors.
$ php -v
PHP 7.0.7 (cli) (built: May 25 2016 18:36:46) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies
    with Xdebug v2.5.0-dev, Copyright (c) 2002-2016, by Derick Rethans
````

**Comment by GrahamCampbell** [comment]
Well, more spefically, too new a version of phpunit's code coverage package.

**Comment by Ocramius** [comment]
Works if running with a `"phpunit/php-code-coverage": "^3.0.0"` constraint, but that will require a lower version of PHPUnit as well (up to `5.3.*`)

**Comment by padraic** [comment]
Merged #183 - let me know if there's any lingering issues.
